cmake/PCTHeader.h

```diff
@@ -0,0 +1,17 @@
+/*=========================================================================
+ *
+ *  Copyright RTK Consortium
```

Comment:
I'd rather use "Centre National de la Recherche Scientifique" for the copyrights, as in the license file